### DISCUSSION TITLE
App doesn’t retain context after tapping “Open” in browser

### DISCUSSION THREAD

**Discussion by marlonjames71** [community/community#5280] [Mobile]
If I'm looking at something on GitHub with the browser on my phone, a banner shows up at the top asking me to open in the GitHub app.

**What actually happens:**
After tapping the "Open" button, the app just opens up to the main screens as if you normally opened the app up from the icon on your Home Screen. 

**Expected:**
The app should open up to the content you were looking at in the browser only in the app instead.

The disconnect here is that it feels like the browser is basically telling me to  open the content I'm looking at in the app because I'll have a better experience, but that content never shows in the actual app.

**Work Arounds:**
I either just keep looking at the content in the browser, or I'll have to manually go into the app and search for what I was looking at.

That's honestly a hassle though as there is some back and forth. Also, the banner at the top in the browser seems pretty pointless.

Thanks for hearing me out. I hope this is something that can be fixed.

Cheers!

- Marlonhttps://user-images.githubusercontent.com/48145861/130731120-ebfafc09-79c6-4381-aa67-8e872e3756db.MOV

**Comment by elisealix22** [comment]
Hi @marlonjames71 , thank you so much for this report and for including a video! I created an internal issue to track this. If you have any other bug reports or feature requests, please keep them coming!

**Comment by marlonjames71** [comment]
Sounds good. Thanks!

∿ M⋀rl◌n ∿

On August 30, 2021, Michael ***@***.***> wrote:
> Hi @marlonjames71 <https://github.com/marlonjames71> , thank you so
> much for this report and for including a video! I created an internal
> issue to track this. If you have any other bug reports or feature
> requests, please keep them coming!
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/github/feedback/discussions/5280#discussioncomment-
1254583>, or unsubscribe <https://github.com/notifications/unsubscribe-
auth/ALPKLRKMFUMMZJMPOP6RMCTT7OL65ANCNFSM5CYID33A>.
